reports/usage-analytics/index.md

```diff
@@ -0,0 +1,185 @@
+
+Configure the following model properties (any other properties can use their default values):
+
+- **Model Id**: Skuid_Usage
+- **Data Source Type**: Salesforce
+- **Data Source**: Salesforce
+- **Salesforce Object Name**: Skuid_Usage__c
+- **Query on Page Load**: False
+- **Max ## of records**: 1 
+
+![alt_text](images/image3.png "image_tooltip")
+
+Then add the following model fields:
+
+- CreatedById
+- CreatedDate
+- Name (or _Page Name_ if you renamed the field)
+- Tracked_Action_Category__c
+- Tracked_Action_Name__c
+
+### Step 4: Create a 'Usage Snapshot' action sequence
+
+Next, [create a reusable action sequence](https://docs.skuid.com/latest/en/skuid/action-framework/action-sequences/#reusable-action-sequences) on your page and name it something like 'Usage Snapshot'. **This action sequence is what will be used to create the analytics records.** By setting up inputs, you can reuse this sequence anywhere you wish to record a tracked action.
+
+- **Note**: Similar to what we did with the Skuid_Usage model, it's best to add this action sequence to your master page. That way this action sequence will be available to use on every child page. 
+
+First, configure the basic information for the action sequence:
+
+- **Type**: Reusable Action Sequence
+- **Sequence Name**: Usage Snapshot
+
+You can also add a _description_ if needed.
+
+Next, click **the Inputs tab** and add the following inputs: 
+
+1. An input to record the tracked action category
+   - **Name**: trackedActionCategory
+   - **Type**: Value
+2. An input to record the tracked action name
+   - **Name**: trackedActionName
+   - **Type**: Value
+
+Now return to the Actions tab to create the sequence by adding the following actions (any action properties not listed below can use their default values):
+
+- **Action 1**: Remove all rows from Model
+  - **Models to Empty**: Skuid_Usage
+- **Action 2**: Create new row(s)
+  - **Model**: Skuid_Usage
+    - **Update Field 1**: Record the category of action
+      - **Field**: Tracked_Action_Category__c
+      - **Field Value Source**: Single Specified Value
+      - **Value**: {{$Input.trackedActionCategory}}
+    - **Update Field 2**: Record the specific action
+      - **Field**: Tracked_Action_Name__c
+      - **Field Value Source**: Single Specified Value
+      - **Value**: {{$Input.trackedActionName}}
+    - **Update Field 3**: Records the page name where the action happened.
+      - **Field**: Name _or_ Page Name
+      - **Field Value Source**: Page/URL parameter value
+      - **Parameter**: page
+    - **Update Field 4**: Record the date that the user performed the action.
+      - **Field**: CreatedDate
+      - **Field Value Source**: Single specified value
+      - **Value**: Today
+    - **Update Field 4**: Records the user's ID.
```

Comment:
`CreatedById` is not writeable --- it will be populated automatically by Salesforce. This will do nothing, and is unnecessary.